### DISCUSSION TITLE
Feedback on "for you" (ability to block)

### DISCUSSION THREAD

**Discussion by mattwelke** [community/community#13390] [Other Feature Feedback, Questions, & Ideas]
I recently tried the "for you" feed. I noticed it had an option for providing feedback which brought me here, so I figured I'd mention an issue I ran into.

The person at the very top of the feed was somebody I'm specifically trying to avoid exposure to because of a very negative experience dealing with them online. I have already unfollowed them on GitHub, blocked them on Twitter, etc. But because GitHub doesn't seem to have a block feature right now, I couldn't do that, and they're showing up in this new feed. They have a yellow star. I'm not sure what that means. Maybe it's because they're prominent in the open source community. Maybe that's why they showed up in my feed, and at the very top.

I wouldn't be comfortable using the feed where it chooses who to display content for unless I have the ability to block people on GitHub.

**Comment by sulfo** [comment]
Would love an ignore or block option as well. Seeing users I'm trying to avoid as well in the feed, and I really don't want to follow what they are doing on GitHub.

Update:
GitHub is currently working on a fix for it, if you have blocked a user: #13161 
However it dose not allow for "soft block" like just ignore a user

**Reply by mattwelke** [reply]
I was actually able to find the "block" feature (didn't know it existed until yesterday) and when I blocked the user, they did disappear from the top of the new feed. The action they performed in the feed was starring a repo. So maybe right now, GitHub is working on preventing users from showing up in the feed for all actions they perform. For when they star repos, it seems to be already implemented (at least if you block them after they show up in the feed).

**Comment by mattwelke** [comment]
Update: I was able to find an existing block user feature, and it worked for me for this use case. See https://github.com/github/feedback/discussions/13390#discussioncomment-2432953.

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
